src/main/java/fr/techad/edc/popover/internal/swing/builder/ContextualTitleComponentBuilderImpl.java

```diff
@@ -1,13 +1,22 @@
 package fr.techad.edc.popover.internal.swing.builder;
 
+import com.google.common.collect.Sets;
+import fr.techad.edc.client.EdcClient;
+import fr.techad.edc.client.internal.TranslationConstants;
```

Comment:
We should never use internal because it's not "visible" as public class. If this is visible, this contants enum should move in client.model.